app/src/components/governance/PageProposal.vue

```diff
@@ -258,19 +141,38 @@ export default {
     noVotes() {
       return BigNumber(this.proposal.tally.total).eq(0)
     },
-    getNextProposalId() {
-      let id = this.getProposalIndex(-1)
-      return id
+    tallyHasValues() {
+      return Object.values(this.proposal.tally)
+        .filter((value) => value !== `Tally`)
+        .find((value) => value)
     },
-    getPrevProposalId() {
-      let id = this.getProposalIndex(1)
-      return id
+    isSecond() {
+      return (
+        this.currentNetwork.network_type === `polkadot` &&
+        this.status.badge === `Deposit Period`
+      )
     },
-  },
-  watch: {
-    // Needed to show data loading component when you are browsing from one proposal to another
-    $route: function () {
-      this.loaded = false
+    participants() {
+      if (
+        this.proposal.detailedVotes.votes &&
+        this.proposal.detailedVotes.votes.length > 0
```

Comment:
this would also trigger when the proposal is in the voting period but no votes are cast yet

Reply:
don't think so. You need at least one vote to trigger this